test/client_test.js

```diff
@@ -177,29 +178,28 @@ suite('api (client)', function() {
       expires, description, scopes,
     });
 
-    await helper.events.waitFor('created');
+    assert.deepEqual(helper.publisher.calls, [{method: 'clientDeleted', clientId: CLIENT_ID},
+      {method: 'clientDeleted', clientId: CLIENT_ID}]);
   };
 
   test('auth.resetAccessToken', async () => {
-    await createTestClient();
```

Comment:
Hi Dustin, sorry for the delay. It runs just fine in my laptop, I removed it because I get this error : 

`Error: client with same clientId already exists, possibly an issue with retry logic or idempotency
----
method:     createClient
errorCode:  RequestConflict
statusCode: 409
time:       2018-04-12T04:11:53.069Z
    at node_modules/taskcluster-client/lib/client.js:318:21
    at <anonymous>
    at process._tickDomainCallback (internal/process/next_tick.js:228:7)`

Reply:
That's pretty weird.  If you look at the top of the file, there's a `cleanup` function there which is invoked before and after each test script, and deletes all clients except `root`.  Perhaps you had that commented out when you were testing?